### PR TITLE
DEVELOPER-2476 - Fixed Fuse documentation issue

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

File: products/fuse/get-started.adoc [hunk 1]
```diff
@@ -4,10 +4,6 @@
 == Prerequisites section title
 Prerequisites &#38; Required Dependencies Checklist
 
-== Prerequisites section
-1. Java SE Development Kit (JDK), we recommend OpenJDK or Oracle JDK.
```
Comment (paulrobinson) [inline products/fuse/get-started.adoc]: I think you still need to leave in this pre-req. Did @catrobson say otherwise?

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

File: products/fuse/get-started.adoc (hunk 1 above)
Comment (Dantheman720) [inline products/fuse/get-started.adoc]: @paulrobinson @catrobson - I think I misunderstood the comments in the JIRA ticket.  I will fix the PR.

File: products/fuse/get-started.adoc [hunk 2]
```diff
@@ -6,7 +6,6 @@ Prerequisites &#38; Required Dependencies Checklist
 
 == Prerequisites section
 1. Java SE Development Kit (JDK), we recommend OpenJDK or Oracle JDK.
-2. JBoss&#174; Developer Studio 9.0
```
Comment (paulrobinson) [inline products/fuse/get-started.adoc]: Sorry, I was wrong. You need to remove all the pre-reqs. Notice that the JDK install is mentioned here: https://github.com/redhat-developer/developers.redhat.com/pull/736/files#diff-4969c2a46f168bb48a422e3abe0cf507R21

Comment (paulrobinson) [inline products/fuse/get-started.adoc]: You should check the other get-started guides to see if this problem is present there too.

**Comment by ollyjshaw** [comment]
docker build

**Comment by ollyjshaw** [comment]
retest this please

**Comment by ollyjshaw** [comment]
retest this please

**Pull request merged**